extensions/cornerstone/src/utils/measurementServiceMappings/utils/getSOPInstanceAttributes.js

```diff
@@ -1,5 +1,36 @@
 import * as cornerstone from '@cornerstonejs/core';
 
+function getDisplaySet({ metadata, displaySetService }) {
```

Comment:
This change probably isn't required, but it makes it less likely to throw an exception on navigation for an over-constrained annotation position.